website/docs/routes/journal.md

```diff
@@ -394,20 +410,8 @@ You can get all short name of a journal from <https://www.nature.com/siteindex>
 
 ## ScienceDirect {#sciencedirect}
 
-### Journal {#sciencedirect-journal}
-
```

Comment:
This subtitle belongs to other feed.

Reply:
Sorry about it, I didn't do a good job of reviewing the content in the changed files.

Reply:
> This subtitle belongs to other feed.
Sorry about it, I didn't do a good job of reviewing the content in the changed files.

Reply:
> This subtitle belongs to other feed.

Sorry about it, I didn't do a good job of reviewing the content in the changed files.